staramr/results/QualityModule.py

```diff
@@ -0,0 +1,219 @@
+                    if line[0] == '>':
+                        if length == 0:
+                            continue
+                        else:
+                            genome_length = genome_length + length
+                            contig_lengths.append(length)
+                            length = 0        
+                    else:
+                        length = length + len(line)
+            contig_lengths.append(length)
+            files_contigs_lengths.append(contig_lengths)
+            files_genomes_lengths.append(genome_length+length)
+        feedback.append(files_contigs_lengths)
+        feedback.append(files_genomes_lengths)
+        return feedback
+
+    def _get_genome_length_feedback(self,files,files_genome_lengths,lb_gsize,ub_gsize):
+        """
+        Goes through the files and determines whether or not they pass the quality metrics for genome length
+        :param files: The files for which we wish to determine whether or not they pass the quality metrics for genome length
+        :param files_genome_lengths: An array where each element is the genome length of the corresponding file
+        :param lb_gsize: The lower bound for the genome size as defined by the user for quality metrics
+        :param ub_gsize: The upper bound for the genome size as defined by the user for quality metrics
+        :return: An array where each element corresponds to the feedback (true or false) for the corresponding file in regards to the
+        genome size quality metric
+        """
+        files_genome_feedback=[]
+        for genome_length in files_genome_lengths:
+            if genome_length >= lb_gsize and genome_length <= ub_gsize:
+                files_genome_feedback.append(True)
+            else:
+                files_genome_feedback.append(False)
+        return files_genome_feedback
+
+    def _get_N50_feedback(self,files_contigs_lengths,files_genome_lengths,minimum_N50):
+        #For information on what N50 is and how it is calculated see https://en.wikipedia.org/wiki/N50,_L50,_and_related_statistics
+        """
+        Goes through the files and determines whether or not they pass the quality metrics for N50 value
+        :param files_contigs_lengths: The lengths of the contigs for the files
+        :param files_genome_lengths: An array where each element is the genome length of the corresponding file
+        :param minimum_N50_value: The minimum N50 value as defined by the user for quality metrics
+        :return: An array where the first element is itself an array where each element is the N50 value for 
+        the corresponding file. The second element is itself an array where each element is the feedback (true or false) 
+        for whether the corresponding file passes the N50 quality metrics
+        """
+        feedback = []
+        file_index =0
+        files_N50=[]
+        N50_feedback = []
+        for files_genome_length in files_genome_lengths:
```

Comment:
Looks like you creating variable files_genome_length but not using it anywhere in the loop. Believe "contig_lengths = files_contigs_lengths[file_index]" will have same value as  "files_genome_length".

No need for "file_index" variable.